### DISCUSSION TITLE
original: error: column "statistics->>'chats'" does not exist

### DISCUSSION THREAD

**Discussion by kostyaavdonin** [sequelize/sequelize#16423] [Help & Questions]
I can't get the sum of the chats variable from jsondb...
        User.findAll({
            attributes: [[sequelize.fn('SUM', sequelize.col("users.statistics->>'chats'")), 'sum']] 
        })
        
            statistics: { // стата
      type: DataTypes.JSONB,
      defaultValue: {
        chats: 0, // скок чатов
        messages: 0, // всего сообщений отправлено
        sticker: 0, // стикеров
        document: 0, // документов
        audio: 0, // аудио
        photo: 0, // фото
        video_note: 0, // видео сообщений
        voice: 0, // голосовых
        video: 0 // видео

      }
    },

**Comment by ephys** [comment]
You seem to be trying to extract JSON. This can be done easily [in v7](https://sequelize.org/docs/v7/querying/raw-queries/#sqlattribute), but you're going to have to use `sequelize.literal` in v6 and write that part of the SQL manually
